### DISCUSSION TITLE
The FOSDEM submissions are open at https://penta.fosdem.org/submission

### DISCUSSION THREAD

**Discussion by pombredanne** [software-composition-analysis/fosdem-2022-devroom#9] [Announcements]
https://fosdem.org/2022/schedule/track/software_composition_and_dependency_management/ has been accepted
AND we can now accept submissions for our devroom at https://penta.fosdem.org/submission

@amottier @maxhbr @cnuel @mcjaeger @kestewart

**Comment by amottier** [comment]
Once #10 is merged we will be all set for promoting the CfP :slightly_smiling_face: 
I just realized that I didn't include a link to [discussions](https://github.com/software-composition-analysis/fosdem-2022-devroom/discussions) in the readme. I'm editing now to add this.
